Trading/Exchange/binance_websocket_feed/binance_websocket.py

```diff
@@ -0,0 +1,44 @@
+#  Drakkar-Software OctoBot-Tentacles
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_trading.exchanges as exchanges
+import cryptofeed.defines as cryptofeed_constants
+from octobot_trading.enums import WebsocketFeeds as Feeds
+
+
+class BinanceCryptofeedWebsocketConnector(exchanges.CryptofeedWebsocketConnector):
+    REQUIRED_ACTIVATED_TENTACLES = []
+    EXCHANGE_FEEDS = {
+        Feeds.TRADES: Feeds.UNSUPPORTED.value,
+        Feeds.KLINE: Feeds.UNSUPPORTED.value,
```

Comment:
```suggestion
        Feeds.KLINE: cryptofeed_constants.CANDLES,
```
with https://github.com/Drakkar-Software/OctoBot-Trading/pull/532 I propose we do it this way :)

Reply:
👍